apps/aechannel/src/aesc_fsm.erl

```diff
@@ -1915,7 +2012,14 @@ check_deposit_signed_msg(#{ channel_id := ChanId
                           , data       := TxBin} = Msg,
                           #data{on_chain_id = ChanId} = Data) ->
     SignedTx = aetx_sign:deserialize_from_binary(TxBin),
-    {ok, SignedTx, log(rcv, ?DEP_SIGNED, Msg, Data)}.
+    case check_type_and_verify_signatures(SignedTx, channel_deposit_tx,
+                                          [other_account(Data),
+                                           my_account(Data)],
```

Comment:
Not terribly important, but one could imagine a `both_accounts(Data)` helper function. ;-)